app/uk/gov/hmrc/customs/declaration/connectors/ApiSubscriptionFieldsConnector.scala

```diff
@@ -17,10 +17,9 @@
 package uk.gov.hmrc.customs.declaration.connectors
```

Comment:
An error with `AuthAction` not in this PR, but has been fixed in exports - please see https://github.com/hmrc/customs-inventory-linking-exports/blob/master/app/uk/gov/hmrc/customs/inventorylinking/export/controllers/actionbuilders/AuthAction.scala#L94 and https://github.com/hmrc/customs-inventory-linking-exports/blob/master/app/uk/gov/hmrc/customs/inventorylinking/export/controllers/actionbuilders/AuthAction.scala#L125
`AuthAction` has been changed so that where we throw exception this has been replaced by a `Left(...)`